src/main/resources/reference.conf

```diff
@@ -56,6 +56,9 @@ ergo {
     # Time window within which a node performs mempool cleanup in between blocks application
     mempoolCleanupDuration = 20s
 
+    # Number of transactions from mempool to be re-broadcasted at each epoch
+    reBroadcastVolume = 200
```

Comment:
Isn't it too much for the default value? E.g. a node has 100 connections, it will receive 20k messages every 2 minutes which may lead to problems.

Reply:
This setting is now decreased down to 20 transactions.

Reply:
finally, to 3